### PR TITLE
pill color changes in mobile-app

### PR DESCRIPTION
**Pull request by SowmyaFyle**
updated pills:
image
image
image

comments fixes:
image
image
image

### CODE REVIEW

File: src/global.scss [hunk 1]
```diff
@@ -1268,3 +1268,31 @@ ion-toggle.toggle-checked::before {
...
+  border: 1px solid $brand-primary;
+  border-radius: 14px;
+  color: $brand-primary;
+  font-size: 12px;
+  font-weight: 500;
+  display: inline-block;
+  line-height: 1;
+  padding: 5px 12px;
+  text-align: center;
+  text-transform: uppercase;
```
Comment (yksolanki9) [inline src/global.scss:1292]: From the ss, it is not working for report and expense states probably bc of angular pipe. Please fix this

**Review by yksolanki9 — CHANGES_REQUESTED** [review]
- Seems you have missed `app-expense-card` component. Please make the changes there as well
- Also in advances all the letters in state pill are capitalized whereas they are in titlecase elsewhere, please confirm how it should be with design and make them consistent

File: src/theme/colors.scss [hunk 2]
```diff
@@ -96,3 +98,98 @@ $payment-processing-color: #47a447;
...
+  'rejected': $rejected-color,
+  'approved': $highlight-green,
+  'payment_pending': $highlight-green,
+  'payment_processing': darken(#5cb85c, 8%),
+  'paid': darken(#5cb85c, 15%),
+  'disabled': $balanced,
+  //state pill in advance requests
+  'pending': #2196f3,
+  'issued': #3d8b3d,
+  //state pill in integration exports
```
[the remark this answers is not in the document]
Comment (SowmyaFyle) [reply src/theme/colors.scss:120]: not require. removed

File: src/global.scss (hunk 1 above)
Comment (SowmyaFyle) [reply src/global.scss:1292]: changed

**Comment by SowmyaFyle** [comment]
> * Seems you have missed `app-expense-card` component. Please make the changes there as well
> * Also in advances all the letters in state pill are capitalized whereas they are in titlecase elsewhere, please confirm how it should be with design and make them consistent

done

**Review by yksolanki9 — APPROVED** [review]

**Review by Julias0 — APPROVED** [review]

**Pull request merged**